index.html

```diff
@@ -42,7 +42,156 @@ <h1>Hey There.<br>I'm Clarence</h1>
+        <a class="see-project" href="#">See project</a>
+    </div>
+    <div class="card-3">
+        <h2>Profesional Art Printing Data</h2>
+        <p>
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required. Has been the industry's standard.
+        </p>
+        <ul class="link-1">
+            <li><a href="#">HTML</a></li>
+            <li><a href="#">Bootstrap</a></li>
+            <li><a href="#">RUBY</a></li>
+        </ul>
+        <a class="see-project" href="#">See project</a>
+    </div>
+    <div class="card-3">
+        <h2>Profesional Art Printing Data</h2>
+        <p>
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required. Has been the industry's standard.
+        </p>
+        <ul class="link-1">
+            <li><a href="#">HTML</a></li>
+            <li><a href="#">Bootstrap</a></li>
+            <li><a href="#">RUBY</a></li>
+        </ul>
+        <a class="see-project" href="#">See project</a>
+    </div>
+    <div class="card-3">
+        <h2>Profesional Art Printing Data</h2>
+        <p>
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required. Has been the industry's standard.
+        </p>
+        <ul class="link-1">
+            <li><a href="#">HTML</a></li>
+            <li><a href="#">Bootstrap</a></li>
+            <li><a href="#">RUBY</a></li>
+        </ul>
+        <a class="see-project" href="#">See project</a>
+    </div>
+    <div class="card-3">
+        <h2>Profesional Art Printing Data</h2>
+        <p>
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required. Has been the industry's standard.
+        </p>
+        <ul class="link-1">
+            <li><a href="#">HTML</a></li>
+            <li><a href="#">Bootstrap</a></li>
+            <li><a href="#">RUBY</a></li>
+        </ul>
+        <a class="see-project" href="#">See project</a>
+    </div>
+    <div class="card-3 last">
+        <h2>Profesional Art Printing Data</h2>
+        <p>
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required. Has been the industry's standard.
+        </p>
+        <ul class="link-1">
+            <li><a href="#">HTML</a></li>
+            <li><a href="#">Bootstrap</a></li>
+            <li><a href="#">RUBY</a></li>
+        </ul>
+        <a class="see-project" href="#">See project</a>
+    </div>
+  </section>
+  <section class="about-myself">
+    <h2>About me</h2>
+    <p>
+        Hello I'm a software developer! I can help you build a product ,
+        feature or website Look through some of my work and experience!
+        If you like what you see and have a project you need coded, don't hesitate to contact me.
+    </p>
+    <a class="see-project button-1" href="#">Get my resume</a>
+    <img src="images/Frame.svg" alt="frame image">
```

Comment:
- [ ] Kindly note that this is supposed to be a background image. Kindly add it using CSS with the *background-image* property. 👍🏾